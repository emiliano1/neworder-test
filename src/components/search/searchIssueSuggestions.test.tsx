import * as React from 'react'
import SearchIssueSuggestions from './searchIssueSuggestions'
import renderer from 'react-test-renderer'
import Issue from '../../models/issue'

it('renders correctly', () => {
  const issues: Issue[] = [{
    id: 1,
    title: 'test 1',
    user: {
        avatarUrl: 'http://example.com/avatar.png',
        url: "http://example.com/user1",
        login: "user1"
    },
    created_at: new Date().toISOString(),
    labels: [{
      title: "test label",
      color: "fff"
    }],
    body: "this is a test issue"
}]
  const tree = renderer.create(<SearchIssueSuggestions issues={issues}/>).toJSON();
  expect(tree).toMatchSnapshot();
})