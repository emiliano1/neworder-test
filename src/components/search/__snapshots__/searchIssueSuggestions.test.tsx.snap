// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="MuiList-root makeStyles-root-1 MuiList-padding"
>
  <div
    aria-disabled={false}
    className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex={0}
  >
    <div
      className="MuiListItemText-root"
    >
      <span
        className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
      >
        test 1
      </span>
    </div>
  </div>
</nav>
`;
